plugins/aea-test-autonomy/aea_test_autonomy/docker/tendermint.py

```diff
@@ -139,6 +141,7 @@ def __create_flask_tendermint_image(self) -> None:
         try:
             self._client.images.get(self.tag)
         except ImageNotFound:
+            # TOFIX - remove.
```

Comment:
this remains to be done still, is it a separate issue or simply forgotten?

Reply:
Separate issue